Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -60,6 +61,22 @@ class TabLocationView: UIView, FeatureFlaggable {
         return true
     }
 
+    private var isFakespotFeatureEnabled: Bool {
+        guard featureFlags.isFeatureEnabled(.fakespotFeature, checking: .buildOnly) else {
+            return false
+        }
+        return true
+    }
```

Comment:
To me, this implies that we're gonna be littering and manually checking this stuff wherever we want to use it.

Would it not be better to have a central place where this is checked and where key components of the feature live? For example, a `FakespotFeatureManager` or something? @thatswinnie I don't know how you're architecting this, but your input here would be valuable, I think, for how you envision the overall feature being built.

But, then, you could have that as a central place from which to operate. And also, because it would have dependencies in the init, it would be really easy to just do a `FakespotFeatureManager().isFeatureEnabled` type of thing, to avoid Singletons. Again, this depends on what the overall structure of the code will look like.

Update: I just saw the `NimbusFakespotFeatureLayer`.  Having, at a minimum, a layer is the general pattern we want to follow with Nimbus: ie, having a layer to take out things from nimbus, so I love seeing this. But that's the only job the layer should have. For example: with Onboarding, the layer exists to feed things to the onboarding manager; with Wallpapers, the layer exists to feed things into the wallpaper manager. So perhaps that's a pattern that we can establish. So same thought applies here.